iosApp/iosApp/Views/Components/MoreTextFieldHL.swift

```diff
@@ -24,12 +24,15 @@ struct MoreTextFieldHL: View {
     var capitalization: Capitalization = .normal
     var autoCorrectDisabled = false
     var textType: UITextContentType? = nil
+    var hlAlignment = TextAlignment.leading
```

Comment:
Da gibt man normal den Typ an und setzt dann dahinter den default Wert in Swift
```suggestion
    var hlAlignment: TextAlignment = .leading
```